frameworks/kafka/src/main/dist/server.properties.mustache

```diff
@@ -27,21 +27,29 @@ broker.id={{POD_INSTANCE_INDEX}}
 #     listeners = security_protocol://host_name:port
 #   EXAMPLE:
 #     listeners = PLAINTEXT://your.host.name:9092
+#
+# Hostname and port the broker will advertise to producers and consumers. If not set,
+# it uses the value for "listeners" if configured.  Otherwise, it will use the value
+# returned from java.net.InetAddress.getCanonicalHostName().
+#
+# advertised.listeners=PLAINTEXT://your.host.name:9092
+
 {{^SECURITY_KERBEROS_ENABLED}}
 {{^KAFKA_ENABLE_TLS}}
-listeners=PLAINTEXT://:{{KAFKA_BROKER_PORT}}
+listeners=PLAINTEXT://{{MESOS_CONTAINER_IP}}:{{KAFKA_BROKER_PORT}}
 port={{KAFKA_BROKER_PORT}}
+advertised.listeners=PLAINTEXT://{{TASK_NAME}}.{{FRAMEWORK_HOST}}:{{KAFKA_BROKER_PORT}}
```

Comment:
Would it make sense to have all the listeners defined here (instead of here *OR* in the TLS section *OR* in the SASL section)?